content/applications/hr/frontdesk.rst

```diff
@@ -0,0 +1,318 @@
+  Click on this line to be taken to a list view of all visitors that are checked-in.
+- :guilabel:`(X) Planned`: This indicates the total number of guests that are planned to arrive that
+  day. Click on this line to be taken to a list view of all visitors that are planned to arrive.
+- :guilabel:`(X) Drinks to serve`: This indicates the total number of drinks that have been
+  requested by checked-in guests, but have not been delivered and marked as served yet. Click on
+  this line to be taken to a list view of all visitors that are waiting for their drink to be
+  served.
+- :guilabel:`Last Check-In`: This indicates how long ago the last check-in took place, in hours
+  and/or minutes.
+- :guilabel:`Additional options menu`: Click this to reveal a drop-down menu, to access either the
+  :guilabel:`Kiosk` in a separate tab, the full list of planned and/or checked-in
+  :guilabel:`Visitors` for the current day, the :guilabel:`Statistics` for that station, or the
+  :guilabel:`Configuration` options for that particular station.
+- :guilabel:`Assign`: The user who is notified when someone checks-in at this station. Add another
+  user by clicking the :guilabel:`Assign` icon and selecting any additional users desired.
+
+.. image:: frontdesk/main-lobby.png
+   :align: center
+   :alt: Frontdesk kiosk card with the stats shown on it.
+
+Click on any station card to navigate to a complete list of all visitors associated with that
+particular station, past, present, and future.
+
+.. _frontdesk/visitors:
+
+Visitors
+========
+
+To access a complete list of visitors that are currently checked in, navigate to
+:menuselection:`Frontdesk application --> Visitors`. The visitors are listed in a list view, with
+the following details that were entered when they checked-in:
+
+- :guilabel:`Name`: The guest's name.
+- :guilabel:`Visitor Company`: The guest's company they represent.
+- :guilabel:`Phone`: The guest's phone number.
+- :guilabel:`Drinks`: The drink the guest requested.
+- :guilabel:`Host`: Who the guest is waiting to see.
+- :guilabel:`CheckIn`: The date and time the guest checked-in.
+- :guilabel:`Checkout`: The date and time the guest checked-out. In the default view, only guests
+  with a :guilabel:`Checked-In` or :guilabel:`Planned` status are visible. Guests with check-out
+  times are only visible if the filter :guilabel:`Today` is not active.
+- :guilabel:`Duration`: The current amount of time the guest has been checked-in for.
+- :guilabel:`Station`: The location of where the guest checked-in.
+- :guilabel:`Status`: The status of the guest. The options are :guilabel:`Checked-In`,
+  :guilabel:`Planned`, :guilabel:`Checked-Out`, or :guilabel:`Cancelled`. In the default view, only
+  guests with a :guilabel:`Checked-In` or :guilabel:`Planned` status are visible.
+- :guilabel:`Email`: The guest's email address.
+- Status Column: When a guest leaves, click the :guilabel:`Check out` button to update the guest's
```

Comment:
```suggestion
- :guilabel:`Status` Column: When a guest leaves, click the :guilabel:`Check out` button to update the guest's
```

Reply:
I don't know why the comment wasn't saved, I know I already explained the reason why I didn't go through with this change- so here it is again. I originally agreed, but then changed my mind. The reason is, there is no text I am quoting on the list. The column header is blank, so that's why Status Column is not capitalized- I am not quoting something on the screen.